changelog.txt

```diff
@@ -0,0 +1,2 @@
+feature - Experimental: add ability to locally emulate HTTPS functions with "firebase serve --only functions". (Run "firebase serve --only functions,hosting" to emulate both functions and hosting.)

```

Comment:
Nit: Let's not use a parenthetical.

    feature - [Experimental] Add ability to locally emulate HTTPS functions with "firebase serve --only functions". Run "firebase serve --only functions,hosting" to serve both functions and hosting.